docs/admin/src/widgets/StringTrimmed.js

```diff
@@ -1,39 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default class StringTrimmedControl extends React.Component {
-  static propTypes = {
-    onChange: PropTypes.func.isRequired,
-    forID: PropTypes.string,
-    value: PropTypes.node,
-    classNameWrapper: PropTypes.string.isRequired,
-    setActiveStyle: PropTypes.func.isRequired,
-    setInactiveStyle: PropTypes.func.isRequired,
-  };
+export default function StringTrimmedControl({
+  onChange,
+  forID,
+  value,
+  classNameWrapper,
+  setActiveStyle,
+  setInactiveStyle,
+}) {
+  const [value, setValue] = React.useState(value || '');
 
-  state = {
-    value: this.props.value || '',
+  const handleChange = (event) => {
+    onChange(event.target.value.trim());
```

Comment:
I think this onChange line should be deleted

and the next line can be:
setValue(event.target.value.trim())

Reply:
Is that safe? Seems like it would change behavior vs the code that's there now, if an `onChange` function is passed in.

Reply:
Sorry, you're right.

 I didn't see it was being passed in and was confused by the onChange inside of the input